core/src/com/unciv/logic/city/CityInfo.kt

```diff
@@ -599,7 +599,7 @@ class CityInfo {
     }
     
     fun canBeDestroyed(): Boolean {
-        return !isCapital() && !isHolyCity()
+        return !isOriginalCapital && !isCapital() && !isHolyCity()
```

Comment:
Hm, this should probably replace the isCapital actually

Reply:
Maybe. You know the codebase better than me. I'm actually sure the behaviour is a regression but haven't yet browsed the histories to look for the break...

Reply:
This is part of an attempt to enable 'capture all original capitals' as a victory condition in the future, so yes, it should replace.
This was written before isOriginalCapital even existed, hence the slight inaccuracy

Reply:
I seem to remember not being able to raze conquered city states no matter what and now I can? `isOriginalCapital` must have already existed in that 'before' picture. Probably some other code changed. But `canBeDestroyed` sounds logical, unequivocal, and if it's to hold to the promise of its name it should respect `isOriginalCapital` methinks...